python/gui/auto_generated/qgsmapcanvas.sip.in

```diff
@@ -70,6 +70,14 @@ empty string before setLayers() calls can be made.
 Gets access to properties used for map rendering
 
 .. versionadded:: 2.4
+%End
+
+    void setTemporalController( QgsTemporalNavigationObject *controller );
```

Comment:
This should be QgsTemporalController, not QgsTemporalNavigationObject

Reply:
This means we should rename QgsTemporalNavigationObject to QgsTemporalController right?

Reply:
No, I mean QgsTemporalNavigationObject should be inheriting QgsTemporalController, and the canvas should accept the base interface class.

Reply:
Ok